api/src/main/java/org/openmrs/module/appointmentscheduling/reporting/data/definition/AppointmentCreatorDataDefinition.java

```diff
@@ -0,0 +1,39 @@
+package org.openmrs.module.appointmentscheduling.reporting.data.definition;
+
+import org.openmrs.module.reporting.common.Localized;
+import org.openmrs.module.reporting.data.BaseDataDefinition;
+import org.openmrs.module.reporting.definition.configuration.ConfigurationPropertyCachingStrategy;
+import org.openmrs.module.reporting.evaluation.caching.Caching;
+
+@Caching(strategy=ConfigurationPropertyCachingStrategy.class)
+@Localized("appointmentscheduling.AppointmentCreatorDataDefinition")
+public class AppointmentCreatorDataDefinition extends BaseDataDefinition implements AppointmentDataDefinition {
+
+    public static final long serialVersionUID = 1L;
+
+    /**
+     * Default Constructor
+     */
+    public AppointmentCreatorDataDefinition() {
+        super();
+    }
+
+    /**
+     * Constructor to populate name only
+     */
+    public AppointmentCreatorDataDefinition(String name) {
+        super(name);
+    }
+
+    //***** INSTANCE METHODS *****
+
+    /**
+     * @see org.openmrs.module.reporting.data.DataDefinition#getDataType()
+     */
+    public Class<?> getDataType() {
+        return String.class;
```

Comment:
This should be "User.class", as the "creator" property we are returning is a User.